src/terra/Factory/EnvironmentFactory.php

```diff
@@ -584,6 +584,24 @@ public function getDrushPort()
     }
 
     /**
+     * Get's the exposed port of the Drupal Console container.
+     *
+     * @return bool|mixed
+     */
+    public function getDrupalConsolePort()
+    {
+        $process = new Process('docker-compose port console 22', $this->getDockerComposePath());
```

Comment:
So, if i understand correctly the 'drush' image is just an SSH daemon, so you're saying we can just leverage the drush image.  Maybe makes sense to rename the drush image to something more generic?